Server/Controller/OpenIdDiscoveryController.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using AuthServer.Shared;
+using AuthServer.Server.Models;
+
+namespace AuthServer.Server.Controllers
+{
+    [ApiController]
+    [Route("/.well-known/openid-configuration")]
+    public class OpenIdDiscoveryController : ControllerBase
+    {
+
+        private readonly ILogger<OpenIdDiscoveryController> _logger;
```

Comment:
Codacy found an issue: [Remove this unread private field '_logger' or refactor the code to use its value.](https://app.codacy.com/gh/LukasReschke/AuthServer/pullRequest?prid=6395991)